tests/rule_based_profiler/data_assistant/test_onboarding_data_assistant.py

```diff
@@ -145,3 +254,390 @@ def test_onboarding_data_assistant_result_batch_id_to_batch_identifier_display_n
+            }
+        ),
+        alt.Tooltip(
+            **{
+                "field": "max_value",
+                "format": ",",
+                "title": "Max Value",
+                "type": AltairDataTypes.QUANTITATIVE.value,
+            }
+        ),
+        alt.Tooltip(
+            **{
+                "field": "strict_min",
+                "format": "",
+                "title": "Strict Min",
+                "type": AltairDataTypes.NOMINAL.value,
+            }
+        ),
+        alt.Tooltip(
+            **{
+                "field": "strict_max",
+                "format": "",
+                "title": "Strict Max",
+                "type": AltairDataTypes.NOMINAL.value,
+            }
+        ),
+        alt.Tooltip(
+            **{
+                "field": "column_distinct_values_count",
+                "format": ",",
+                "title": "Column Distinct Values Count",
+                "type": AltairDataTypes.QUANTITATIVE.value,
+            }
+        ),
+    ]
+
+    single_column_return_chart: alt.LayerChart = plot_result.charts[2]
+    layer_1: alt.Chart = single_column_return_chart.layer[1]
+    actual_tooltip: List[alt.Tooltip] = layer_1.encoding.tooltip
+
+    assert actual_tooltip == expected_tooltip
+
+
+def test_onboarding_data_assistant_plot_descriptive_non_sequential_notebook_execution(
+    bobby_columnar_table_multi_batch_deterministic_data_context,
+):
+    context: DataContext = bobby_columnar_table_multi_batch_deterministic_data_context
+
+    new_cell: str = "data_assistant_result.plot_metrics(sequential=False)"
+
+    run_onboarding_data_assistant_result_jupyter_notebook_with_new_cell(
+        context=context,
+        new_cell=new_cell,
+        implicit=False,
+    )
+
+    run_onboarding_data_assistant_result_jupyter_notebook_with_new_cell(
+        context=context,
+        new_cell=new_cell,
+        implicit=True,
+    )
+
+
+def test_onboarding_data_assistant_plot_descriptive_non_sequential_notebook_execution(
+    bobby_columnar_table_multi_batch_deterministic_data_context,
+):
+    context: DataContext = bobby_columnar_table_multi_batch_deterministic_data_context
+
+    new_cell: str = (
+        "data_assistant_result.plot_expectations_and_metrics(sequential=False)"
+    )
+
+    run_onboarding_data_assistant_result_jupyter_notebook_with_new_cell(
+        context=context,
+        new_cell=new_cell,
+        implicit=False,
+    )
+
+    run_onboarding_data_assistant_result_jupyter_notebook_with_new_cell(
+        context=context,
+        new_cell=new_cell,
+        implicit=True,
+    )
+
+
+def test_onboarding_data_assistant_plot_non_sequential(
+    bobby_onboarding_data_assistant_result: OnboardingDataAssistantResult,
+) -> None:
+    sequential: bool = False
+    plot_metrics_result: PlotResult = (
+        bobby_onboarding_data_assistant_result.plot_metrics(sequential=sequential)
+    )
+
+    assert all([chart.mark == "bar" for chart in plot_metrics_result.charts])
```

Comment:
Nitpick - I think we can do without the list here since `all()` works with any iterator. Performance/memory shouldn't really be an issue but some linters complain about this.

Reply:
✅